docs/tools_scripts.md

```diff
@@ -258,3 +258,57 @@ python tools/deploy_test.py \
 
 - All ONNX models are evaluated with dynamic shape on the datasets and images are preprocessed according to the original config file.
 - This tool is still experimental, and we only support `restorer` for now.
+
+
+### Convert ONNX to TensorRT (experimental)
+We also provide a script to convert [ONNX](https://github.com/onnx/onnx) model to [TensorRT](https://github.com/NVIDIA/TensorRT) format. Besides, we support comparing the output results between ONNX and TensorRT model.
+
+
+```bash
+python tools/onnx2tensorrt.py
+    ${CFG_PATH} \
+    ${MODEL_TYPE} \
+    ${IMAGE_PATH} \
+    ${INPUT_ONNX} \
+    --trt-file ${OUT_TENSORRT} \
+    --max-shape INT INT INT INT \
+    --min-shape INT INT INT INT \
+    --workspace-size INT \
+    --fp16 \
+    --show \
+    --verify \
+    --verbose
+```
+
+Description of arguments:
+
+- `config` : The path of a model config file.
+- `model_type` :The model type of the config file, options: `inpainting`, `mattor`, `restorer`, `synthesizer`.
+- `img_path` : The path to input image file.
+- `onnx_file` : The path to input ONNX file.
+- `--trt-file` : The path of output TensorRT model. If not specified, it will be set to `tmp.trt`.
+- `--max-shape` : Maximum shape of model input.
+- `--min-shape` : Minimum shape of model input.
+- `--workspace-size`: Max workspace size in GiB. If not specified, it will be set to 1 GiB.
+- `--fp16`: Determines whether to export TensorRT with fp16 mode. If not specified, it will be set to `False`.
+- `--show`: Determines whether to show the output of ONNX and TensorRT. If not specified, it will be set to `False`.
+- `--verify`: Determines whether to verify the correctness of an exported model. If not specified, it will be set to `False`.
+- `--verbose`: Determines whether to verbose logging messages while creating TensorRT engine. If not specified, it will be set to `False`.
+
+**Note**: This tool is still experimental. Some customized operators are not supported for now. We only support `restorer` for now. While generating ONNX file of SRCNN, replace 'bicubic' with 'bilinear' in 40th line of mmedit/models/backbones/sr_backbones/srcnn.py. For TensorRT does not support bicubic interpolation by now and final performace will be weaken by about 4%.
```

Comment:
Could add the [link](https://github.com/open-mmlab/mmediting/blob/764e6065e315b7d0033762038fcbf0bb1c570d4d/mmedit/models/backbones/sr_backbones/srcnn.py#L40) for the "40th line" : 

> `https://github.com/open-mmlab/mmediting/blob/764e6065e315b7d0033762038fcbf0bb1c570d4d/mmedit/models/backbones/sr_backbones/srcnn.py#L40`

Reply:
Resolved.